kubert/src/lease.rs

```diff
@@ -300,7 +296,17 @@ impl LeaseManager {
                 }
 
                 // Update the claim and broadcast it to all receivers.
-                claim = self.ensure_claimed(&claimant, &params).await?;
+                match self.ensure_claimed(&claimant, &params).await {
+                    Ok(new_claim) => claim = new_claim,
+                    Err(error) if Self::is_retryable(&error) => {
+                        tracing::debug!(%error, "failed to claim lease, retrying");
+                        // TODO(eliza): this will immediately retry (since the
+                        // expiration should have elapsed)...should we back off
+                        // here?
```

Comment:
Yes, we need to avoid tight looping if this situation isn't resolved immediately.

Reply:
initially i didn't want to include an exponential backoff implementation in this PR because i wasn't sure if it made sense for it to be user-configurable, and i thought i would punt on that. but i'm happy to add it now --- do you think backoff parameters ought to be configurable, or should we just pick something?